src/NonCollectMemoryManager.cpp

```diff
@@ -92,7 +127,14 @@ bool NonCollectMemoryManager::initializeHeap(size_t heapSize, size_t maxSize)
 
 void NonCollectMemoryManager::growHeap()
 {
-    uint8_t* heap = static_cast<uint8_t*>( std::malloc(m_heapSize) );
+    TMemoryManagerEvent event;
+    event.heapInfo.usedHeapSizeBeforeCollect = m_usedHeaps.size()*m_heapSize/1024;
+    m_memoryInfo.heapSize += m_usedHeaps.size()*m_heapSize*(m_memoryInfo.heapIncreaseTimer.get<TMicrosec>().toInt()/1000000.0) / 1024.0;
+    m_memoryInfo.heapIncreaseTimer.start();
+    m_memoryInfo.collectionsCount++;
+    event.eventName = "GC";
+    event.begin = m_memoryInfo.timer.get<TSec>();
+    uint8_t* heap = static_cast<uint8_t*>(std::malloc(m_heapSize));
```

Comment:
Copy this line from original source (`uint8_t* heap = static_cast<uint8_t*>( std::malloc(m_heapSize) );`)